typegate/src/config.ts

```diff
@@ -39,6 +39,8 @@ const schema = {
     }
     return bytes;
   }),
+  timer_max_timeout_ms: z.number().positive().max(60000), // 1min
+  timer_destroy_ressources: z.boolean().default(true),
```

Comment:
Let's have all default at the same place: `.default(true)` shall already be taken care by line 86